<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyDialog</class>
 <widget class="QDialog" name="keyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>unlock and mount a luks volume</string>
  </property>
  <widget class="QPushButton" name="pbOpen">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>key</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;key</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbKeyFile">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>key &amp;from keyfile</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPlugIn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;plugin</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEditKey">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenKeyFile">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mount point</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenMountPoint">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMountPoint">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditKey</tabstop>
  <tabstop>rbKey</tabstop>
  <tabstop>rbKeyFile</tabstop>
  <tabstop>pbOpenKeyFile</tabstop>
  <tabstop>pbOpen</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
